ДЗ. TypeScript. Анотація типів та примітивні типи.ts

```diff
@@ -70,44 +75,47 @@ class Group {
 
 class Student {
   grades = {};
-  attendance = [];
+  attendance: boolean[] = [];
+  firstName: string;
+  lastName: string;
+  birthYear: number;
 
-  constructor(firstName, lastName, birthYear) {
+  constructor(firstName: string, lastName: string, birthYear: number) {
     this.firstName = firstName;
     this.lastName = lastName;
     this.birthYear = birthYear;
   }
 
-  get fullName() {
+  get fullName(): string {
     return `${this.lastName} ${this.firstName}`;
   }
 
-  set fullName(value) {
+  set fullName(value: string) {
     [this.lastName, this.firstName] = value.split(" ");
   }
 
-  get age() {
+  get age(): number {
     return new Date().getFullYear() - this.birthYear;
   }
 
-  setGrade(subject, grade) {
+  setGrade(subject: string, grade: number) {

```

Comment:
Please, do not forget to explicitly add type annotation for the value that returns the function.